docs/usage/server.md

```diff
@@ -1,37 +1,40 @@
 # Unity Catalog Server
+
 This is the open source version of the Unity Catalog server implementation. It can be started locally for local usage (e.g. for testing) without any Databricks account.
 
 ## Running the Server
 
 The server can be started by issuing the below command from the project root directory:
-```sh  
+
+```sh
 bin/start-uc-server
 ```
 
 ## Configuration
 
 The server config file is at the location `etc/conf/server.properties` (relative to the project root).
 
-- `server.env`: The environment in which the server is running. This can be set to `dev` or `test`. When 
- set to `test` the server will instantiate an empty in-memory h2 database for storing metadata.
- If set to `dev`, the server will use the file `etc/db/h2db.mv.db` as the metadata
- store. Any changes made to the metadata will be persisted in this file.
+- `server.env`: The environment in which the server is running. This can be set to `dev` or `test`. When
+  set to `test` the server will instantiate an empty in-memory h2 database for storing metadata.
```

Comment:
Can we just put this all on the same line?  I don't like chopping up sentences on multiple lines.

Reply:
Yup, good catch!